inst/templates/study_schema.R

```diff
@@ -0,0 +1,28 @@
+#' Study Schema for {{ study_name }}
+#'
+#' To source the study schema in a chunk in your Rmarkdown report:
+#' source("R/study_schema.R")
+#'
+#' @return a kable table with the study schema
+#' @export
+#'
+#' @examples
+study_schema <- function() {
+
+  caption <- "{{ study_name }} study schema."
+
+  schema_table <- tibble::tribble(
+    ~Group, ~`Sample Size`, ~`Week 10`, ~`Week 20`,
+    "Group A", 10, "Dose A", "Dose A",
+    "Group B", 10, "Dose B", "Dose B"
+    )
+
+  schema_table %>%
+    knitr::kable(
+      format = output_type,
+      caption = caption,
+      booktabs = TRUE,
+      linesep = ""
+    ) %>%
+    kableExtra::kable_styling(latex_options = c("hold_position"))
```

Comment:
Not sure about this one since it's a template function. When someone sets up a new project with `create_visc_project()`, this is created and then the user can go and edit it as needed to use within their project.